client/src/containers/AddressPage.tsx

```diff
@@ -160,7 +160,16 @@ export default class AddressPage extends Component<AddressPageProps, State> {
               <div className="float-left">
                 <h1 className="primary">
                   <Trans>
-                    PORTFOLIO: Your search address is associated with <u>{assocAddrs.length}</u>{" "}
+                    PORTFOLIO: Your search address is associated with{" "}
+                    <Link
+                      to={routes.portfolio}
+                      tabIndex={this.props.currentTab === 2 ? -1 : 0}
```

Comment:
The way I'm reading this, it says if you're already on the portfolio tab, go left one tab; otherwise stay on the same page. That seems backwards to me - am I reading it wrong?

Reply:
So this was all just copied from the link for clicking directly on the portfolio tab. My understanding of `tabIndex` is that it's only about tab-navigation, and not what shown visually - that part is handled below by changing css class `"active"` on and off based on `currentTab` values. For a link to be tab focusable it needs to have `tabIndex=0`, but when a tab is already active we want to exclude so set `tabIndex=-1`. 

https://user-images.githubusercontent.com/16906516/165582510-9da15f14-09d8-4f73-a79c-774124fd0f5e.mov